backend/src/src-cvesearch/src/main/java/com/bosch/osmi/sw360/cvesearch/datasource/CveSearchApi.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) Bosch Software Innovations GmbH 2016.
+ * Part of the SW360 Portal Project.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package com.bosch.osmi.sw360.cvesearch.datasource;
+
+import java.io.IOException;
+import java.util.List;
+
+public interface CveSearchApi {
+
+    List<CveSearchData> search(String vendor, String product) throws IOException;
+    List<CveSearchData> cvefor(String cpe) throws IOException;
```

Comment:
That naming is taken from the rest api we talk to. (see https://github.com/cve-search/cve-search/blob/master/web/minimal-web.py#L182)